src/main/java/com/conveyal/datatools/manager/jobs/FeedUpdater.java

```diff
@@ -166,21 +173,17 @@ private void updatePublishedFeedVersion(String feedId, FeedSource feedSource) {
             // feed that failed processing by RTD.
             Optional<FeedVersion> lastPublishedVersionCandidate = versions
                 .stream()
-                .min(Comparator.comparing(v -> v.published, Comparator.nullsLast(Comparator.reverseOrder())));
+                .min(Comparator.comparing(v -> v.sentToExternalPublisher, Comparator.nullsLast(Comparator.reverseOrder())));
             if (lastPublishedVersionCandidate.isPresent()) {
                 FeedVersion publishedVersion = lastPublishedVersionCandidate.get();
-                if (publishedVersion.published == null) {
+                if (publishedVersion.sentToExternalPublisher == null) {
                     LOG.warn("Not updating published version for {} (published date is null)", feedId);
```

Comment:
This log statement could be updated to note not being sent to the external publisher.